src/main/java/codeu/model/store/basic/ActivityStore.java

```diff
@@ -0,0 +1,52 @@
+package codeu.model.store.basic;
+
+import codeu.model.data.Activity;
+import codeu.model.store.persistence.PersistentDataStoreException;
```

Comment:
What the differences between
codeu.model.store.persistence.PersistentDataStoreException; and codeu.model.store.persistence.PersistentStorageAgent; ??????

Reply:
PersistentDataStoreException is just an Exception. You throw it when an unexpected situation occurs to either try to recover from that situation or terminate the program. The PersistentStorageAgent is an interface for all the PersistanceLogic we have